src/web-style.md

```diff
@@ -42,41 +24,12 @@ The js-beautify tool can be handy for indenting HTML as well.
 
 * Make it compliant with RECESS. [See explanation below](#recess).
 
-Tools
------
-
-### </a>JSHint
-
-Use JSHint <http://jshint.com/> to check for errors and make the
-source compatible with our coding conventions.  The jshint command is
-provided by sugar-build.
+### JavaScript
 
-To check JavaScript code:
+* The JavaScript style guide is now in its own document [here](#https://github.com/sugarlabs/sugar-docs/blob/master/src/javascript-style.md)
```

Comment:
No need to say "now in its own document". Just say "here"

Reply:
Thank you!